src/app/app-extras.module.ts

```diff
@@ -28,10 +33,20 @@ const treeModuleForRoot = TreeModule.forRoot();
     treeModuleForRoot
   ],
   exports: [
+    SkyAngularTreeModule,
     SkyAppLinkModule,
+    SkyDocsToolsModule,
     SkyDropdownModule,
-    SkyAngularTreeModule,
     TreeModule
+  ],
+  providers: [
```

Comment:
Are these docs changes not in the master branch already?

Reply:
Nope. Should we release a 3 version with that before we release 4?